base/set.jl

```diff
@@ -1,5 +1,42 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    Set{T} <: AbstractSet{T}
+
+`Set`s are mutable containers that provide fast membership testing.
+
+`Set`s have efficient implementations of set operations such as `in`, `union` and `intersect`.
+Elements in a `Set` are unique, as determined by the elements' definitions of
+`isequal` and `hash`.
+The order of elements in a `Set` is an implementation detail and cannot be relied on.
+
+See also: [`AbstractSet`](@ref), [`BitSet`](@ref), [`Dict`](@ref),
+[`push!`](@ref), [`empty!`](@ref), [`union!`](@ref), [`in`](@ref), [`isequal`](@ref)
+
+# Examples
+```jldoctest filter = r"^\\S.+"
+julia> s = Set("aaBca")
```

Comment:
```suggestion
julia> s = Set("aaBca"); push!(s, 'b')
```
If you are worried it to be too long, I guess you can connect these together without losing any vital information.